code_map.py

```diff
@@ -1,296 +1,346 @@
 import sublime
 import sublime_plugin
-import codecs
 import os
 import shutil
 import sys
 import zipfile
 from os import path
 from sublime import Region
-import socket
-import subprocess
-import errno
-from socket import error as socket_error
+from importlib.machinery import SourceFileLoader
+import CodeMap.code_map_support as Mapper
+from CodeMap.code_map_support import NavigateCodeMap as Nav
 
```

Comment:
some stuff has been moved to support script, mappers and some cumbersome but little relevant functions